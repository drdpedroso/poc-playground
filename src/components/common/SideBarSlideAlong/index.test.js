import React from 'react'
import {SideBarSlideAlong} from './index'
import renderer from 'react-test-renderer'

describe('SideBarSlideAlong', () => {
  it('should render correctly', () => {
    const component = renderer.create(<SideBarSlideAlong />)
    expect(component).toMatchSnapshot()
  })
})
