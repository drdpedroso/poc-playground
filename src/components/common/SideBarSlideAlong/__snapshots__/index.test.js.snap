// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarSlideAlong should render correctly 1`] = `
<div>
  <div
    className="ui segment pushable"
  >
    <div
      className="ui inverted vertical labeled icon ui slide along left thin visible sidebar menu"
    >
      <a
        className="item"
        onClick={[Function]}
      >
        
        PEDIDOS
      </a>
      <a
        className="item"
        onClick={[Function]}
      >
        
        HISTÃ“RICO DE PEDIDOS
      </a>
      <a
        className="item"
        onClick={[Function]}
      >
        
        ESTOQUE DE PRODUTOS
      </a>
      <a
        className="item"
        onClick={[Function]}
      >
        
        MINHA CONTA
      </a>
      <a
        className="item"
        onClick={[Function]}
      >
        
        SAIR
      </a>
    </div>
    <div
      className="pusher"
    >
      <div
        className="ui basic segment"
      >
        <div
          className="sc-bdVaJa jReomQ"
        />
      </div>
    </div>
  </div>
</div>
`;
